app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_likes_counter
+    like.increment!(:likes_counter)
+  end
+end
```

Comment:
Dear @Ngufor-emmanuella, As the previous code reviewer mentioned Likes belong to a post, and the counter should be updated for a specific post. Please make sure to use the `post.increment!(:likes_counter)`, that way the like counter for the post will be updated.